### PR TITLE
tkey-ssh-agent: Adjustments

### PR DESCRIPTION
**Pull request by johan-carlberg**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----
Install man page.
Added `--uss` and `--pinentry` flag when running service.

### CODE REVIEW

**Comment by johan-carlberg** [comment]
I did add a commit that makes this work on Linux. Do I need to take any more action to continue the progress?

**Comment by johan-carlberg** [comment]
What do we need to do to make progress on this pull request? @SMillerDev

**Review by SMillerDev — COMMENTED** [review]
@dawidd6 you made adjustments to the service for this right?

**Comment by iMichka** [comment]
@johan-carlberg can you rebase this please? And can someone please post the errors / describe the issue we are encountering. The build logs are gone and I can't force push to your repo @johan-carlberg

**Comment by dehanj** [comment]
@iMichka I did a rebase on master, not sure about any errors/issues though..

**Comment by iMichka** [comment]
I thought there were errors, thus the hold up. But the build is green.
I am a little bit confused about the comments above: @SMillerDev are we waiting on something from @dawidd6 ?

@dehanj can you also squash this to a single commit?

**Comment by SMillerDev** [comment]
Well, I didn't think it would work like this so I was asking Dawidd to confirm since I remember a previous issue with different services per OS.

**Comment by dehanj** [comment]
> @dehanj can you also squash this to a single commit?

Squashed @iMichka

**Comment by iMichka** [comment]
Ok so the assumption is that the conditional service code might not work as expected?
Can we test this?

@dehanj @johan-carlberg I guess the code above works on macOS, right?
How can I test this on Linux to make sure nothing gets broken? I am not really a brew service user, so I don't know what I should check.

File: Formula/tkey-ssh-agent.rb [hunk 1]
```diff
@@ -55,7 +56,15 @@ def caveats
...
-    run [opt_bin/"tkey-ssh-agent", "--agent-socket", var/"run/tkey-ssh-agent.sock"]
+    args = [
+      opt_bin/"tkey-ssh-agent",
+      "--agent-socket",
+      var/"run/tkey-ssh-agent.sock",
+      "--uss",
+    ]
+    args << "--pinentry" if OS.mac?
+    args << (HOMEBREW_PREFIX/"bin/pinentry-mac") if OS.mac?
+    run args
```
Comment (dawidd6) [inline Formula/tkey-ssh-agent.rb:67]: It should be done like this: 
https://github.com/Homebrew/homebrew-core/blob/623ee367cf1ed16548bead54ecc47abc2187921f/Formula/podman.rb#L163

**Comment by johan-carlberg** [comment]
> Ok so the assumption is that the conditional service code might not work as expected? Can we test this?
> 
> @dehanj @johan-carlberg I guess the code above works on macOS, right? How can I test this on Linux to make sure nothing gets broken? I am not really a brew service user, so I don't know what I should check.

@iMichka: The way to test this is to start the service, and verify that `$HOMEBREW_PREFIX/var/run/tkey-ssh-agent.sock` is created.

File: Formula/tkey-ssh-agent.rb (hunk 1 above)
Comment (johan-carlberg) [reply Formula/tkey-ssh-agent.rb:67]: Ah, I wasn't aware of that syntax. I'll attempt to use that instead.

Comment (johan-carlberg) [reply Formula/tkey-ssh-agent.rb]: Done.

**Review by SMillerDev — DISMISSED** [review]

**Comment by dawidd6** [comment]
> ==> brew style homebrew/core
==> FAILED
Full style homebrew/core output
  /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/tkey-ssh-agent.rb:59:119: C: Layout/LineLength: Line is too long. [150/118]
      run macos: [opt_bin/"tkey-ssh-agent", "--agent-socket", var/"run/tkey-ssh-agent.sock", "--uss", "--pinentry", HOMEBREW_PREFIX/"bin/pinentry-mac"],
                                                                                                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  
heh

**Review by dawidd6 — DISMISSED** [review]

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**